ragna/deploy/_ui/js.py

```diff
@@ -1,20 +1,10 @@
-from .styles import CONFIG_MODAL_MAX_HEIGHT, CONFIG_MODAL_MIN_HEIGHT
-
-
-# Reset modal size
-def reset_modal_size(width, height):
+def set_modal_size(width, height):
     return f"""
     document.getElementById("pn-Modal").style.setProperty("--dialog-height","{height}px", "important")
     document.getElementById("pn-Modal").style.setProperty("--dialog-width","{width}px", "important")
     """
 
 
-JS_TOGGLE_CARD = f"""
```

Comment:
Removing because no longer works on `main` (see previous comments where a new `param` object has been introduced for the upload container height).